src/index.tsx

```diff
@@ -6,14 +6,26 @@ import { Provider } from 'react-redux';
 import './index.css';
 import App from './components/App';
 import configureStore from './configureStore';
+import { actions as apiActions } from './reducers/api';
 
 const store = configureStore();
 
+const rootElement = document.getElementById('root') as HTMLElement;
+const authToken = rootElement ? rootElement.dataset.authToken : null;
+
+if (
+  authToken &&
+  authToken.length &&
+  authToken !== process.env.REACT_APP_AUTH_TOKEN_PLACEHOLDER
```

Comment:
Does this case mean that the placeholder was not replaced? If so, this should throw an error because it means someone is running the app incorrectly.